README.md

```diff
@@ -6,10 +6,10 @@
          * [Building InfiniBand Kubernetes Binary](#building-infiniband-kubernetes-binary)
          * [Building Subnet Manager Plugins](#building-subnet-manager-plugins)
          * [Building Container Image](#building-container-image)
-      * [Configuration reference](#configuration-reference)
+      * [Configuration Reference](#configuration-reference)
       * [Deploying](#deploying)
 
-# IB-Kubernetes
+# InfiniBand Kubernetes
 
 InfiniBand Kubernetes cooperate with [Mellanox InfiniBand SR-IOV CNI](https://github.com/Mellanox/ib-sriov-cni) and 
```

Comment:
a reword suggestion:
Infiniband kubernetes provides a daemon, 'ib-kubernetes', that works in conjuction with Mellanox Infiniband SR-IOV CNI......... , it acts on kubernetes Pod object changes(Create/Update/Delete), reading the Pod's network annotation and fetching its corresponding network CRD and ......

Reply:
changed